scripts/download-chromium.js

```diff
@@ -98,4 +99,52 @@ exports.downloadForPlatform = async function downloadForPlatform(electron_platfo
     getChromeVersion()
   );
   await download(platform, arch, revision, directory);
+  setPermissions(directory);
 };
+
+async function setPermissions(directory) {
```

Comment:
the error handling on this function seems really off, I would suggest to bubble up the error and handle with a single try/catch logic.

Reply:
It's what happens when you spend hours figuring out why a CI build is failing, and then write a quick hack to try and fix it 😄